packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -68,6 +69,13 @@ export default function globalReferenceToImport(
     return fileForIdentifier[identifier];
   }
 
+  function searchForNamedExports(filename) {
+    if (filename.endsWith('.coffee'))
+      return false;
+    const namedExports = j(fs.readFileSync(filename).toString())
```

Comment:
What about cases where the word `export` is used inside a string like `'Click to export products'`. The grep has no feasible way of preventing finds like that.